setup.py

```diff
@@ -43,6 +43,11 @@ def parse_requires():
 with open("README.md") as f:
     long_desc = f.read()
 
+
+extensions = [
+    Extension('uarray._uarray', sources=['uarray/_uarray_dispatch.cxx'], language='c++')
```

Comment:
This might fix it. 😄 

I also implemented your suggestions during the meeting in #169.

```suggestion
    Extension('uarray._uarray', sources=['uarray/_uarray_dispatch.cxx'], language='c++11')
```